src/bootstrap/bootstrap.py

```diff
@@ -1035,6 +1035,10 @@ def check_vendored_status(self):
                 eprint('       Run `cargo vendor {}` to initialize the '
                       'vendor directory.'.format(sync_dirs))
                 eprint('Alternatively, use the pre-vendored `rustc-src` dist component.')
+                eprint('You can download it from '
+                    'https://forge.rust-lang.org/infra/other-installation-methods.html#source-code')
+                eprint('and then place the vendor directory from the archive in the root')
+                eprint('of the rust project.')
```

Comment:
Wouldn't it make more sense to use the rustc-src dist component as the source to build rather than copying the vendor directory to a git checkout or rust source obtained in some other way without the vendor directory?

Reply:
I agree

Reply:
> I agree

In some (rare I guess) cases, they may still prefer to copy (if they want to push some changes).

Reply:
It might be my lack of experience in developing rust, but is it really a better option? I imagine copying the vendor directory to my git-backed rust codebase would be easier. If I wanted to develop, I would have the repository synced, set up with git, and an easy way of pushing the changes. If I worked from the downloaded source, there wouldn't be all that. Those changes would have to be moved to git-backed repo, or that directory would have to get synced with git anyway.

Having said that, I agree this option should be added, yet I would consider it secondary in order.

Reply:
As I mentioned above, both makes sense to me. I am fine with either way.